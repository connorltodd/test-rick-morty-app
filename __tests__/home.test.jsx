import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import Home from "../app/page";

describe("Home Page", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
