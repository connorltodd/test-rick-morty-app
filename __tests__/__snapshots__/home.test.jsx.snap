// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders homepage unchanged 1`] = `
<div>
  <main
    class="flex justify-center items-center pt-20 flex-col gap-20"
  >
    <img
      alt="rick and morty"
      data-nimg="1"
      decoding="async"
      height="200"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
      style="color: transparent;"
      width="200"
    />
    <h1
      class="text-lg w-[40rem] text-center"
    >
      Welcome to the Rick and Morty API Explorer, where you can get more information on all of your favourite Characters, Episodes and Locations.
       
    </h1>
    <ul
      class="flex justify-center gap-20"
    >
      <li
        class="bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 border border-green-700 rounded"
      >
        <a
          href="/characters"
        >
          View Characters
        </a>
      </li>
      <li
        class="bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 border border-green-700 rounded"
      >
        <a
          href="/locations"
        >
          View Locations
        </a>
      </li>
      <li
        class="bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 border border-green-700 rounded"
      >
        <a
          href="/episodes"
        >
          View Episodes
        </a>
      </li>
    </ul>
  </main>
</div>
`;
